docs/source/api/fpsemi-examples.rst

```diff
@@ -0,0 +1,154 @@
+.. Copyright (c) 2022, J. D. Mitchell
+
+   Distributed under the terms of the GPL license version 3.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+   This file only exists because Breathe always displays all members when
+   documenting a namespace, and this is nicer for now.
+
+.. cpp:namespace:: libsemigroups
+
+Examples
+--------------------
+
+Defined in ``present.hpp`` and ``knuth-bendix.hpp``.
```

Comment:
I'm not sure that this line or the next paragraph is really what you meant, looks more like it's copy pasta ;)